R/translate-sql-base.r

```diff
@@ -167,7 +167,7 @@ base_scalar <- sql_translator(
                       )},
   str_detect      = function(string, pattern){
```

Comment:
Can you please add a comment here pointing to the documentation fo INSTR?

Reply:
I have links to string functions for the different databases at the beginning of the section were we added the `stringr` functions: https://github.com/tidyverse/dbplyr/blob/81419cab7d831e5a9d7e4279a4f893fa5750e8a1/R/translate-sql-base.r#L154-L160 
Does this suffice?

Reply:
Oh yeah, that's sufficient.